rhel.Dockerfile

```diff
@@ -0,0 +1,129 @@
+#    cd /tmp/root-local && sudo tar cvzf root-local.tgz lib/ bin/ && cd - && mv -f /tmp/root-local/root-local.tgz . && sudo rm -fr /tmp/root-local/
+# 3. then add it to dist-git so it's part of this repo
+#    rhpkg new-sources root-local.tgz 
+
+# built in Brew, use tarball in lookaside cache
+# COPY root-local.tgz /tmp/root-local.tgz
+
+# NOTE: uncomment for local build. Must also set full registry path in FROM to registry.redhat.io or registry.access.redhat.com
+# enable rhel 7 or 8 content sets (from Brew) to resolve jq as rpm
+COPY content_sets_epel7.repo /etc/yum.repos.d/
+
+RUN microdnf install -y findutils bash wget yum gzip tar jq python3-six python3-pip && microdnf -y clean all && \
+    # install yq (depends on jq and pyyaml - if jq and pyyaml not already installed, this will try to compile it)
+    if [[ -f /tmp/root-local.tgz ]]; then mkdir -p /root/.local && tar xf /tmp/root-local.tgz -C /root/.local/ && rm -fr /tmp/root-local.tgz; fi && \
+    /usr/bin/pip3.6 install --user yq jsonschema && \
+    # could be installed in /opt/app-root/src/.local/bin or /root/.local/bin
+    for d in /opt/app-root/src/.local /root/.local; do \
+      if [[ -d ${d} ]]; then \
+        cp ${d}/bin/yq ${d}/bin/jsonschema /usr/local/bin/; \
+        pushd ${d}/lib/python3.6/site-packages/ >/dev/null; \
+          cp -r PyYAML* xmltodict* yaml* yq* jsonschema* /usr/lib/python3.6/site-packages/; \
+        popd >/dev/null; \
+      fi; \
+    done && \
+    chmod +x /usr/local/bin/yq && \
+    ln -s /usr/bin/python3.6 /usr/bin/python
+
+# for debugging only
+# RUN microdnf install -y util-linux && whereis python pip jq yq && python --version && jq --version && yq --version
+
+################# 
+# PHASE TWO: configure registry image
+#################
+
+COPY ./scripts/*.sh ./scripts/meta.yaml.schema /build/
+COPY /v3 /build/v3
+WORKDIR /build/
+
+# if only including the /latest/ plugins, apply this line to remove them from builder
+# RUN rm -fr $(find /build/v3 -name 'meta.yaml' | grep -v "/latest/")
+
+RUN ./check_plugins_location.sh v3 && \
+    ./set_plugin_dates.sh v3 && \
+    ./check_plugins_viewer_mandatory_fields.sh v3 && \
+    ./ensure_latest_exists.sh && \
+    ./index.sh v3 > /build/v3/plugins/index.json && \
+    chmod -c -R g+rwX /build
+
+################# 
+# PHASE THREE: create ubi8-minimal image with httpd
+################# 
+
+# Build registry, copying meta.yamls and index.json from builder
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/rhscl/httpd-24-rhel7
+FROM registry.access.redhat.com/rhscl/httpd-24-rhel7:2.4-104 AS registry
+USER 0
+
+# BEGIN these steps might not be required
+RUN sed -i /etc/httpd/conf/httpd.conf \
+    -e "s,Listen 80,Listen 8080," \
+    -e "s,logs/error_log,/dev/stderr," \
+    -e "s,logs/access_log,/dev/stdout," \
+    -e "s,AllowOverride None,AllowOverride All," && \
+    chmod a+rwX /etc/httpd/conf /run/httpd
+EXPOSE 80
```

Comment:
Is it necessary to expose port 80? We're rewriting the conf to listen on 8080.

AFAIK binding to ports <1000 will fail on OpenShift as it requires root.

Reply:
yes we don't care on which port there is a listener

Reply:
This was in the existing [Dockerfile.rhel](https://github.com/eclipse/che-plugin-registry/blob/master/Dockerfile.rhel#L32), but if it's not needed, cool. Interestingly there's nothing in the existing `registry.centos.org/centos/httpd-24-centos7`-based Dockerfile that exposes any ports... 80, 8080, 443, 8443, etc. 

So... is it exposed by default in the upstream httpd-24-centos7 image?

Reply:
`docker inspect registry.centos.org/centos/httpd-24-centos7` gives me posts 8080 and 8443 as exposed. Same thing for `registry.access.redhat.com/rhscl/httpd-24-rhel7:2.4-104`.

Reply:
Port 80 removed.